.github/scripts/releases.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from github import Github
+
+# How to use this script:
+# Set the following env vars:
+#   * GH_RELEASE_TOKEN
+#   * LAMBDA_RUNTIME (optional)
+#   * INITCONTAINER_RUNTIME (optional)
+#   * AGENT_VERSION
+#   * DRY_RUN (optional) set to 1 to run the flow without creating the release tag.
+#
+# Lambda runtime options:
+# nodejs
+# python
+# ruby
+# dotnet
+#
+# Initcontainer runtime options:
+# nodejs
+# python
+# ruby
+# java
+# dotnet
```

Comment:
Why don't you make those a constant, so we don't have to set those as env vars? Presumably each repo will have a separate copy of this script.

Reply:
Well...I was still thinking of making it shared at this point (even though I realize I haven't done that in this PR). I guess TBD on that.